tests/e2e/data/services/test_bq_datasets.py

```diff
@@ -34,27 +38,47 @@ def file_path(path):
 
 
 class TestBQUserDataset(unittest.TestCase):
+    """This test suite needs the ENV variable USERURL pointing to a working DO API in "tests/e2e/secret.json".
+    DO API must have the user/apikey mapping set to get access to the user's DO Project in GCP.
+    """
+
+    def setUp(self):
+        if (os.environ.get('APIKEY') is None
```

Comment:
I'd flip this if/else
```
if os.environment.get('APIKEY') and os.environment.get('USERNAME') and os.environment.get('USERURL')
```

beware that `USERNAME` is a totally legit env variable in unix/linux systems (bsd?) and it may not match CARTO world.

Reply:
Regarding env variable `USERNAME`, it comes from `secrets.json` and I just using it as it is.

Reply:
:man_facepalming: